grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java

```diff
@@ -155,6 +155,15 @@ static void startEmbeddedCassandra() {
         }
     }
 
+    static String randomKeyspace(){
+        if (usingOrientDB()) {
+            return "memory";
```

Comment:
I'm sure you told me this was a limitation of orientdb. You can already see it in `GraphContext` class.